source/common/tracing/zipkin/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,193 @@
+#include "common/tracing/zipkin/zipkin_tracer_impl.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/http/headers.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+#include "common/tracing/http_tracer_impl.h"
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+
+#include "spdlog/spdlog.h"
```

Comment:
If I remove that include statement the code does not compile because of this:
```
throw EnvoyException(fmt::format("{} collector cluster is not defined on cluster manager level",
                                     config.getString("collector_cluster")));
```
As you can see, I am actually using `fmt::format()`. Do you prefer that I do not use `fmt::format` in favor of plain string concatenation.?